core/CronArchive.php

```diff
@@ -235,6 +240,8 @@ public function __construct($processNewSegmentsFrom = null, LoggerInterface $log
         $this->periodIdsToLabels = array_flip(Piwik::$idPeriods);
 
         $this->rawLogDao = new RawLogDao();
+
+        $this->cliMultiRequestParser = new RequestParser($this->makeCliMulti()->supportsAsync());
```

Comment:
@diosmosis did you maybe forget to commit this file?

Reply:
added